### ISSUE TITLE
Ethereum streaming transaction

### ISSUE THREAD

**Issue by shivamgupta01** [corpetty/py-etherscan-api#45]
How can I get hold of Ethereum streaming transaction ? I cannot find any API which will give me the all the transaction happening in Ethereum or Bitcoin.

**Comment by corpetty** [comment]
Yeah I don't think etherscan, and by extension this repo, doesn't provide that.  You might want to look into something like Google's Bigquery (https://cloud.google.com/blog/products/data-analytics/ethereum-bigquery-public-dataset-smart-contract-analytics) or etherslurp (http://www.ethslurp.com/index.html).  If you want all transactions, it might be with your best interest to just run a full-node.

**Comment by shivamgupta01** [comment]
Thanks for updating that, I am trying to run a full node for bitcoin using bitcoin core module and bitcoin - cli, I am not able to find the resource to fetch the transactions. I am trying to work out my way with zmq provided by bitcoin core. Let's see how far I go with that. Please feel free to add this thread with any relevant blog/link you know about achieving that. Cheers.
